docs/DeveloperGuide.md

```diff
@@ -207,6 +207,31 @@ The following activity diagram summarizes what happens when a teacher executes a
     2. Only can set the class to a date at most 1 week away.
   
 
+### Schedule List Feature
```

Comment:
A link to this section should be added to the table of content under `## implementation` section